content/registry/python-asyncpg.md

```diff
@@ -0,0 +1,12 @@
+---
+title: asyncpg-opentracing
+registryType: instrumentation
+tags:
+    - python
+    - opentracing
```

Comment:
I recommend adding asyncpg as a tag, so that it can be searched

Reply:
@yurishkuro Thank you. Done.